api/src/test/java/org/openmrs/util/LiquibaseVersionFinderTest.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.*;
```

Comment:
We generally discourage using wild card imports

Reply:
fixed